src/handlers/event.handler.ts

```
@@ -0,0 +1,68 @@
+      recurrenceEndDate: data.eventData.recurrenceEndDate,
+      recurrencePattern: data.eventData.recurrencePattern,
+      autoEnroll: data.eventData.autoEnroll,
+      registrationStartDate: data.eventData.registrationStartDate,
+      registrationEndDate: data.eventData.registrationEndDate,
+      createdBy: data.eventData.createdBy,
+      updatedBy: data.eventData.updatedBy,
+      eventDetailId: savedEventDetails.eventDetailId,
+    };
+    const savedEvent = await this.dbService.saveEventData(eventData);
+
+    const eventRepertation = data.eventRepetitionData.map(async (eventRepetitionData) => {
```

ESLint

[error] 'eventRepertation' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)